inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.15/src/test/java/org/apache/inlong/sort/tests/utils/FlinkContainerTestEnv.java

```diff
@@ -165,8 +134,8 @@ public void submitSQLJob(String sqlFile, Path... jars)
         ExecResult execResult =
                 jobManager.execInContainer("bash", "-c", String.join(" ", commands));
         LOG.info(execResult.getStdout());
-        LOG.error(execResult.getStderr());
```

Comment:
the error might be useful for debugging, why change the place?

Reply:
I put the code for printing logs into the following if statement.Beacause if There were error in running, the execResult.getExitCode() will not equal 0,So, It will still print logs.

![image](https://github.com/apache/inlong/assets/58425449/e35eb845-7b6c-4d41-b114-cfed2a77bbaf)